ut/m0ub.c

```diff
@@ -106,6 +107,7 @@ static void ub_add(const struct ub_args *args)
 	m0_ub_set_add(&m0_thread_ub);
 //	m0_ub_set_add(&m0_rpc_ub);
 //XXX_BE_DB	m0_ub_set_add(&m0_parity_math_ub);
```

Comment:
`m0_parity_math_ub ` is not enabled.
How did you do the test?

Reply:
Enabled.